build.gradle

```diff
@@ -271,6 +275,39 @@ afterEvaluate {
 ext["itemis.mps.gradle.ant.defaultScriptArgs"] = defaultScriptArgs
 ext["itemis.mps.gradle.ant.defaultScriptClasspath"] = buildScriptClasspath
 
+task resolved_diagram_dependencies(type: Copy) {
```

Comment:
I think we should keep the createResolveTask() invocations below and remove these methods.

Reply:
I was thinking the same, but was not sure whey they were changed in the first place, so I kept them...

Reply:
`resolved_diagram_dependencies(type: Copy)` was of type copy, is it save to be changed to `Sync`?